sonar-html-plugin/src/test/resources/checks/TableWithoutHeaderCheck.html

```diff
@@ -0,0 +1,63 @@
+<table> <!-- Non-Compliant -->
+
+</table>
+
+<table> <!-- Compliant -->
+  <th></th>
+</table>
+
+<table> <!-- Compliant -->
+  <th></th>
+  <table></table> <!-- Non-Compliant -->
+</table>
+
+<table> <!-- Non-Compliant -->
+  <table> <!-- Compliant -->
+    <th></th>
+  </table>
+</table>
+
+<table> <!-- Compliant -->
+  <th></th>
+  <table> <!-- Compliant -->
+    <th></th>
+  </table>
+</table>
+
+<th>
+  
+</th>
+
+<table role="presentation">
+
+</table>
+
+<table role="None">
+
+</table>
+
+<table role="dummy"> <!-- Non-Compliant -->
+
+</table>
+
+<table role="presentation">
+  <table></table>
+</table>
+
+<table aria-hidden="true">
+
+</table>
+
+<table aria-hidden="false"> <!-- Non-Compliant -->
+  
+</table>
+
+<table aria-hidden="true">
+  <table></table>
+</table>
+
+<table aria-hidden="true">
+  <table aria-hidden="false"></table>
```

Comment:
I'm not sure about this. But wouldn't setting `aria-hidden` attribute explicitely to `false` here override the parent aria-hidden attribute ?

Reply:
According to the [Accessibility Developer Guide](https://www.accessibility-developer-guide.com/examples/sensible-aria-usage/hidden/), it will not be overridden.